core/src/main/java/org/infinispan/remoting/rpc/RpcManagerImpl.java

```diff
@@ -433,8 +433,8 @@ private void registerXsiteReplicationTime(long durationMillis) {
       syncXSiteReplicationTime.record(durationMillis);
    }
 
-   private void registerAsyncXSiteReplicationTime(long sendTime, String siteNameIgnored, Throwable throwable) {
-      long durationMillis = timeService.timeDuration(sendTime, TimeUnit.MILLISECONDS);
+   private void registerAsyncXSiteReplicationTime(long sendTimeNanos, String siteNameIgnored, Throwable throwable) {
+      long durationMillis = timeService.timeDuration(sendTimeNanos, TimeUnit.MILLISECONDS);
```

Comment:
Would you mind also changing the names of the parameters in `TimeService.timeDuration()` @pruivo?

Reply:
is it worth to change the name for the `TimeService` (it is documented in the interface the time units)? 
I only changed in the methods because there is no way to know what is the time units. You had to look up for the usage to know it is actually implementing the `XSiteAsyncAckListener` interface.

Reply:
IMO you already know the time unit here, because `timeDuration` only works with nanos, it doesn't matter how `XSiteAsyncAckListener` is defined.

Except of course we don't read the javadoc all the time, so it's not obvious `timeDuration` only works with nanos, and having the unit in the parameter name helps.

Reply:
updated the parameters name in `TimeService` methods